### PR TITLE
Add new examples to README

### PR DESCRIPTION
**Pull request by Andre601**
Thank you for contributing! Please check the following things before submiting your PR:

Required:
- [x] I have read and followed the [contributing guidelines](https://github.com/Readme-Workflows/recent-activity/blob/main/.github/CONTRIBUTING.md)

If necessary:
- [x] I have updated the README and documentation.
- [ ] I have updated the ChangeLog with the changes I have made.

> Added new examples for the README. Please only merge this PR once the actual features have been properly released.

Closes #

---------------------------------------------------------------------

**Note:** we will close your PR without comment if you do not check the required boxes above and provide ALL requested information.

### CODE REVIEW

**Comment by abhijoshi2k** [comment]
Looks good @Andre601 . Thanks for improving the README.
@PuneetGopinath You have any suggestion?

**Comment by PuneetGopinath** [comment]
> Looks good @Andre601 . Thanks for improving the README.
> @PuneetGopinath You have any suggestion?

I don't have any suggestion, you can merge this pr.

**Comment by abhijoshi2k** [comment]
@Andre601 Can you add a link to a page having info about how to create profile readme? (for beginners)
Add it at the top. 
https://abhijoshi2k.medium.com/how-to-create-a-readme-for-your-github-profile-95e0fc474f49

**Comment by PuneetGopinath** [comment]
> @Andre601 Can you add a link to a page having info about how to create profile readme? (for beginners)
> Add it at the top.
> https://abhijoshi2k.medium.com/how-to-create-a-readme-for-your-github-profile-95e0fc474f49
> This will give basic idea to beginners.

Awesome article bro👌👌

@Andre601 just add
```
## For beginners

If you are a beginner, then see [this guide at medium](https://abhijoshi2k.medium.com/how-to-create-a-readme-for-your-github-profile-95e0fc474f49), Guide written by @abhijoshi2k 
```

in the top of the readme

**Comment by Andre601** [comment]
> Awesome article bro👌👌
> 
> @Andre601 just add
> 
> ```
> ## For beginners
> 
> If you are a beginner, then see [this guide at medium](https://abhijoshi2k.medium.com/how-to-create-a-readme-for-your-github-profile-95e0fc474f49), Guide written by @abhijoshi2k 
> ```
> 
> in the top of the readme

Not so sure about this...
Especially in terms of placement. I changed the text and placement a little bit.

File: README.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ Current activities include:
 
 Other activities can't be displayed since they aren't parts of what GitHub's API returns as public events.
 
+### Setup a Profile README
+
+If you're a beginner, then [here is an article](https://abhijoshi2k.medium.com/how-to-create-a-readme-for-your-github-profile-95e0fc474f49) made by [@abhijoshi2k](https://github.com/abhijoshi2k) that explains how you can create and manage your own Profile README.
+
+The official documentation about Profile READMEs can be found [here](https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/managing-your-profile-readme).
```
Comment (PuneetGopinath) [inline README.md:23]: I think `official github documentation` would be better

**Review by PuneetGopinath — CHANGES_REQUESTED** [review]

**Review by abhijoshi2k — APPROVED** [review]
Looks good.
Should we merge this? @PuneetGopinath

**Comment by PuneetGopinath** [comment]
> Looks good.
> Should we merge this? @PuneetGopinath

I requested some changes

**Review by PuneetGopinath — APPROVED** [review]

**Pull request merged**